igcollect/linux_disklatency.py

```diff
@@ -0,0 +1,87 @@
+#!/usr/bin/env python3
+"""igcollect - Linux Disk I/O latency
+
+Copyright (c) 2019 InnoGames GmbH
+"""
+
+from argparse import ArgumentParser
+from subprocess import check_output
+from time import time
+import sys
+from libvirt import (
+    open,
+    libvirtError,
+)
+
+def parse_args():
+
+    parser = ArgumentParser()
+
+    parser.add_argument(
+        '--prefix', dest='prefix', type=str, default='None',
+        help='Prefix to Graphite')
+
+    parser.add_argument(
+        '--storagepool', dest='storagepool', type=str, default='None',
+        help='Storage Pool')
+
+    parser.add_argument(
+        '--storagevol', dest='storagevol', type=str, default='None',
+        help='Storage Volume')
+
+    return parser.parse_args()
+
+def main():
+    URI = 'qemu:///system'
+    args = parse_args()
+    template = args.prefix + '.{}.{} {} ' + str(int(time()))
+
+    try:
+        conn = open(URI)
+    except libvirtError as e:
+        print('Failed to open connection to qemu:///system: {}'.format(e))
+        exit(1)
+
+    try:
+        strgpool = conn.storagePoolLookupByName(args.storagepool)
+    except libvirtError as e:
+        print('Failed to find {} pool: {}'.format(args.storagepool, e))
+        exit(1)
+
+    try:
+        stgvol = strgpool.storageVolLookupByName(args.storagevol)
+    except libvirtError as e:
+        print("An exception has occured while opening the volume: {} {"
+              "}".format(args.storagevol, e))
+        exit(1)
+
+    stgvolpath = stgvol.path()
+
+    output_read = check_output(['/usr/bin/sudo', '/usr/bin/ioping', '-BD',
+                                '-c10', '-i0.1', stgvolpath],
+                               universal_newlines=True)
+
+    output_write = check_output(['/usr/bin/sudo', '/usr/bin/ioping', '-BDWWW',
+                                 '-c10', '-i0.1', stgvolpath],
+                                universal_newlines=True)
```

Comment:
While this is pep8 compliant, we usually prefer the style:
```python
output_read = check_output(
    ['/usr/bin/sudo', '/usr/bin/ioping', '-BD', '-c10', '-i0.1', stgvolpath],
    universal_newlines=True
)
```

Reply:
It doesn't fit in one line as it exceeds 80 chars. I changed it a bit more